term.mw

```diff
@@ -0,0 +1,101 @@
+include stdlib.mw
+
+:: term_write retry
+    local text = pop()
+    term.write(text)
+;;
+
+:: term_scroll retry
+    local y = pop()
+    term.scroll(y)
+;;
+
+:: term_getCursorPos retry
+    local x, y = term.getCursorPos()
+    push(x)
+    push(y)
+;;
+
+:: term_setCursorPos retry
+    local x = pop()
+    local y = pop()
+    term.setCursorPos(x, y)   
+;;

```

Comment:
The order is inconsistent. With the current definition getting the position and then setting the position to the gotten position moves the cursor.

Reply:
Moonwalk currently doesn't have a rule for what order arguments and return values should go in for lua function wrappers, so how about we make them go in the same order they'd appear in lua? (right being top of the stack)

Reply:
reversed some stuff